README.md

```diff
@@ -4,26 +4,53 @@ Anokha 2024 TechFest server.
 
 ## Installation
 
-1. Install Node.js and npm
-2. Install mysql.
+1. Install `Node.js` and `npm`
+2. Install `mysql`.
 3. Create Databases.
 
-```
+```sql
 CREATE DATABASE anokha;
 CREATE DATABASE anokha_transactions;
 ```
 
 4. Install dependencies.
 
-```
+```bash
 npm i
 ```
 
-5. Configure ENV variables .env.local
+5. Configure ENV variables. Create `.env.local` file in the root directory of the project using the following template.
 
-6. Run the server.
+```env
+DB_HOST = '<MYSQL_HOST_NAME>'
+DB_USER = '<MYSQL_USER_NAME>'
+DB_PASS = '<MYSQL_PASSWORD>'
+
+DB_NAME = 'anokha'
+TXN_DB_NAME = 'anokha_transactions'
+
+MAILER_SERVICE = 'Outlook365'
+MAILER_HOST = 'smtp.office365.com'
+MAILER_PORT = 587
```

Comment:
Should we really gave these public?

Reply:
No secrets are revealed. Whatever is already publicly available on the internet and this repo is only used.